cmd/agent/gui/agent.go

```diff
@@ -54,35 +53,33 @@ func ping(w http.ResponseWriter, r *http.Request) {
 }
 
 // Sends the current agent status
-func getStatus(w http.ResponseWriter, r *http.Request, invAgent inventoryagent.Component) {
+func getStatus(w http.ResponseWriter, r *http.Request, invAgent inventoryagent.Component, statusComp statusComponent.Component) {
 	statusType := mux.Vars(r)["type"]
 
 	var (
-		stats map[string]interface{}
-		err   error
+		err      error
+		response []byte
 	)
-	if statusType == "collector" {
-		stats = collector.GetStatusInfo()
+	verbose := r.URL.Query().Get("verbose") == "true"
+	if statusType == statusComponent.CollectorSection {
+		response, err = statusComp.GetStatusBySection(statusComponent.CollectorSection, "html", verbose)
+		if err != nil {
+			log.Errorf("Error getting status: " + err.Error())
+			w.Write([]byte("Error getting status: " + err.Error()))
+			return
+		}
 	} else {
-		verbose := r.URL.Query().Get("verbose") == "true"
-		stats, err = status.GetStatus(verbose, invAgent)
-	}
-
-	if err != nil {
-		log.Errorf("Error getting status: " + err.Error())
-		w.Write([]byte("Error getting status: " + err.Error()))
-		return
-	}
+		response, err = statusComp.GetStatus("html", verbose, statusComponent.CollectorSection)
```

Comment:
For the GUI the HTML views are split into two sections: 
- Collector 
- Everything except collector 

Since the rendering portion is handled in the status component, we need a way to skip sections to maintain the GUI as it is today.